admin/javascript/src/components/hidden-field/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class HiddenFieldComponent extends SilverStripeComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  render() {
+    return (
+      <div className="field hidden">
+        <input {...this.getInputProps()} />
```

Comment:
Oh, id is set to props.name. That makes it harder. I still feel like a callout to a seperate method is unneeded though.